pkg/console/controllers/advancedclustermanagement/managedclusters/controller.go

```diff
@@ -81,7 +80,6 @@ func NewManagedClusterController(
 	ctrl := &ManagedClusterController{
 		operatorClient:       operatorClient,
 		operatorConfigClient: operatorConfigClient,
-		featureGateClient:    configClient.FeatureGates(),
```

Comment:
Lets remove the `configClient` as well from the `NewManagedClusterController` constructor's args